packages/data/src/components/use-dispatch/use-dispatch.js

```diff
@@ -3,25 +3,28 @@
  */
 import useRegistry from '../registry-provider/use-registry';
 
+/** @typedef {import('./types').WPDataStoreDefinition} WPDataStoreDefinition */
+
 /**
  * A custom react hook returning the current registry dispatch actions creators.
  *
  * Note: The component using this hook must be within the context of a
  * RegistryProvider.
  *
- * @param {string} [storeName]  Optionally provide the name of the store from
- *                             which to retrieve action creators. If not
- *                             provided, the registry.dispatch function is
- *                             returned instead.
+ * @param {string|WPDataStoreDefinition} [storeNameOrDefinition] Optionally provide the name of the
```

Comment:
I forgot to rename these, they're actually named `WPDataStore` now

Reply:
see, I will fix it, thanks for catching it 👍